server/leafnode.go

```diff
@@ -499,7 +515,7 @@ func (s *Server) createLeafNode(conn net.Conn, remote *leafNodeCfg) *client {
 		// FIXME(dlc) - Make this resolve at startup.
```

Comment:
If this is at startup we can remove, but if server starts and then can fail at a later time still need to update this IMO.